tests/std/tests/P0660R10_jthread_and_cv_any/test.cpp

```diff
@@ -151,13 +151,13 @@ int main() {
         });
     }
 
+    constexpr auto infinity = chrono::steady_clock::time_point::max();
     { // ditto without the cancellation this would deadlock
         jthread worker([](stop_token token) {
             mutex m;
             condition_variable_any cv;
             unique_lock lck{m};
-            assert(cv.wait_until(lck, move(token), chrono::steady_clock::time_point::max(), [] { return false; })
-                   == false);
+            assert(cv.wait_until(lck, move(token), infinity, [] { return false; }) == false);

```

Comment:
There are compiler errors here - this occurs within a stateless lambda (line 156 `jthread worker([](stop_token token) {`) so we can't access `infinity` outside:

```
C:\a\1\s\tests\std\tests\P0660R10_jthread_and_cv_any\test.cpp(160): error C3493: 'infinity' cannot be implicitly captured because no default capture mode has been specified
```